src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
I fetched this commit locally and did some tests. It seems that removing this casting does not break the build. Can you confirm, @david-driscoll?